### PR TITLE
tests: kernel: add simple ramfunc test

### PR DESCRIPTION
**Pull request by ioannisg**
This commit contributes a simple test for
the ARM RAMFUNC feature.

Signed-off-by: Ioannis Glaropoulos <Ioannis.Glaropoulos@nordicsemi.no>

### CODE REVIEW

File: tests/kernel/arm_ramfunc/prj.conf [hunk 1]
```diff
@@ -0,0 +1,2 @@
+CONFIG_ZTEST=y
+CONFIG_USERSPACE=y
```
Comment (agansari) [inline tests/kernel/arm_ramfunc/prj.conf]: Please add empty line at eof.

**Review by agansari — CHANGES_REQUESTED** [review]
Looks good, needs a minor change.

File: tests/kernel/arm_ramfunc/prj.conf (hunk 1 above)
Comment (ioannisg) [reply tests/kernel/arm_ramfunc/prj.conf]: Will fix, thanks.

**Comment by ioannisg** [comment]
> 
> 
> Looks good, needs a minor change.

Fixed that, @agansari , thanks.

**Review by wentongwu — APPROVED** [review]

**Review by agansari — APPROVED** [review]

**Pull request merged**